client/src/api/API.res

```diff
@@ -261,6 +261,39 @@ let saveTest = (m: model): cmd =>
     x,
   ))
 
+let fetchServerBuildHash = (m: model): cmd => {
+  let url = "https://editor.darklang.com/latest-backend-build-hash"
+
+  let request = Tea.Http.request({
+    method: "POST",
+    headers: list{clientVersionHeader(m), Header("X-CSRF-Token", m.csrfToken)},
+    url: url,
+    body: Web.XMLHttpRequest.EmptyBody,
+    expect: Tea.Http.expectStringResponse(Decoders.wrapExpect(Json.Decode.string)),
+    timeout: None,
+    withCredentials: false,
+  })
```

Comment:
This was largely copied from the one other place where we reference editor.darklang.com - `sendPresence`. It may be worth creating a helper method? (or wait for rule of 3)